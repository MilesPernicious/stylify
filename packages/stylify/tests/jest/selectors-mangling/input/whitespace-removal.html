<div class="
	color:blue
	margin-left:24px
	[a]{color:blue}
">
</div>
