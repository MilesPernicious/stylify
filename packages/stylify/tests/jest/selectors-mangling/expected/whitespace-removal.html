<div class=" b c a ">
</div>
